src/arch/evm/SpokeUtils.ts

```diff
@@ -33,6 +33,7 @@ export function populateV3Relay(
   spokePool: Contract,
   deposit: Omit<Deposit, "messageHash" | "fromLiteChain" | "toLiteChain"> & {
     recipient: EvmAddress;
+    outputToken: EvmAddress;
```

Comment:
After preparing the SVM side it occurred to me we missed narrowing on `outputToken` in the EVM implementation.